package.json

```
@@ -25,7 +25,9 @@
   "license": "MIT",
   "electronVersion": "12.2.3",
   "resolutions": {
-    "es5-ext": "https://github.com/pulsar-edit/es5-ext#169f6ae9b2675675269a0ba265f83c29c7b56244"
+    "es5-ext": "https://github.com/pulsar-edit/es5-ext#169f6ae9b2675675269a0ba265f83c29c7b56244",
+    "superstring": "github:pulsar-edit/superstring",
+    "text-buffer/superstring": "github:pulsar-edit/superstring"
```

Would like this to include exact commit SHAs:

```suggestion
    "superstring": "github:pulsar-edit/superstring#de97b496663fce40050bf2d66e1466ccfbd00943",
    "text-buffer/superstring": "github:pulsar-edit/superstring#de97b496663fce40050bf2d66e1466ccfbd00943"
```